<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectUploadDialog</class>
 <widget class="QDialog" name="ProjectUploadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>工程上传</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="editAddress"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>设备地址：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>上传位置</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>上传文件</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>工程密码：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="btnSelectPath">
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="btnOK">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="editPath"/>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="editPassword">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
